pkg/virt-controller/watch/vmi.go

```diff
@@ -1282,9 +1300,22 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 		}
 
 		if vmiSpecIfaces, vmiSpecNets, dynamicIfacesExist := network.CalculateInterfacesAndNetworksForMultusAnnotationUpdate(vmi); dynamicIfacesExist {
-			if err := c.updateMultusAnnotation(vmi.Namespace, vmiSpecIfaces, vmiSpecNets, pod); err != nil {
+			errMessage := fmt.Errorf("failed to hot{un}plug network interfaces for vmi [%s/%s]", vmi.GetNamespace(), vmi.GetName())
```

Comment:
Why was the error message extracted?

Reply:
so i can use errMessage twice
https://github.com/kubevirt/kubevirt/pull/11410/files/281b90bea6d4abe3adb8c304a743d4fb0b40337a..4c6a0c884374db30a86da95efd30e3c6fe7726ad#diff-16487fff595728eb5eeec01225a2203756ecad25d90e90d5feb13a9ff08df7ffR1311
lines 1311, 1318